tests/spec/test-world-api-service.js

```diff
@@ -396,6 +396,18 @@
                 expect(body.maxUsers).to.be.equal(5);
             });
 
+            it('should pass the userIds if specificed in options', function () {
+                createWorldAdapter({ group: 'group-name' }).autoAssign({ userIds: ['0', '1', '2', '3', '4'] });
+
+                var req = server.requests.pop();
+                var body = JSON.parse(req.requestBody);
```

Comment:
You can just do `expect(body.userIds).to.eql(['0', '1', '2', '4'])` to be more concise.

Also, since userids won't be numbers, use strings like 'a', 'b', etc. Doesn't practically matter of course, just a minor tweak to be more precise.